core/Archive/ArchiveInvalidator.php

```diff
@@ -566,9 +566,9 @@ public function applyScheduledReArchiving()
         $reArchiveList = new ReArchiveList($this->logger);
         $items = $reArchiveList->getAll();
 
-        foreach ($items as $entry) {
+        foreach ($items as $item) {
```

Comment:
btw for such tricky fixes might be good to add a test case if possible as it otherwise might regress again easily